src/python/com/illinoistriangle/bot/outline_bot.py

```diff
@@ -55,7 +58,8 @@ def get_channels(client):
   return _channel_map
 
 
-channel_map = get_channels(sc)
+channel_map = get_channels(bot)
+allowed_channel_ids = [channel_id for channel_id in channel_map if channel_map[channel_id]['name'] in allowed_channels]
```

Comment:
`channel_map[channel_id]['name'] in allowed_channels` is a bug here because 
`allowed_channels = 'test_x'` is a string, instead of `set/list`
```
allowed_channels = ['test_x']
```
should do

Reply:
right now it is a tuple of length one -- there is a comma at the end of that line

Reply:
ah...please definitely make that more clear:
```
allowed_channels = ('test_x', )
```